src/screens/wallet-claim/components/wallet-claim-status.component.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import { Recover } from './claim-status/wallet-recover.component';
+import { NoAccess } from './claim-status/wallet-no-access.component';
+
+export const ActionsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 8rem;
+`;
+
+export const ClaimStaus = () => {
+  return (
+    <ActionsContainer>
+      {/* conditional rendering */}
+      {/* <Recover /> */}
```

Comment:
I think it's better to divide the Recover component into 2. Let's call it `Recovered` and `Recovering` so that we can do conditional rendering here itself and dispay either `Recovered`, `Recovering`  or `NoAccess`. This will be more readable. What do you think?

Reply:
yeah, that will be more readable. will divide the component into 2.